ddtrace/utils/runtime.py

```diff
@@ -0,0 +1,5 @@
+import uuid
+
+
+def generate_runtime_id():
+    return str(uuid.uuid4())
```

Comment:
```suggestion
    return uuid.uuid4().hex
```

This is basically just `str(uuid.uuid4()).replace('-', '')`, might be nicer to remove the `-`

Not really a strong opinion here, happy to keep this, especially if it works just fine. Your call.